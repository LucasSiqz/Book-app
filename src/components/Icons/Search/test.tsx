import { render } from 'utils/test-utils'

import Search from '.'

describe('<SearchIcon />', () => {
  it('should render correctly', () => {
    const { container } = render(<Search />)

    expect(container.firstChild).toMatchSnapshot()
  })
})
