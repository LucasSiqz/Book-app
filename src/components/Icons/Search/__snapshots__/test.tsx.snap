// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<SearchIcon /> should render correctly 1`] = `
<svg
  fill="none"
  height="16"
  viewBox="0 0 16 16"
  width="16"
  xmlns="http://www.w3.org/2000/svg"
>
  <path
    clip-rule="evenodd"
    d="M14.6667 6.66668C14.6667 9.6122 12.2789 12 9.33333 12C6.38781 12 4 9.6122 4 6.66668C4 3.72116 6.38781 1.33334 9.33333 1.33334C12.2789 1.33334 14.6667 3.72116 14.6667 6.66668ZM5.33333 6.66668C5.33333 8.87582 7.12419 10.6667 9.33333 10.6667C11.5425 10.6667 13.3333 8.87582 13.3333 6.66668C13.3333 4.45754 11.5425 2.66668 9.33333 2.66668C7.12419 2.66668 5.33333 4.45754 5.33333 6.66668Z"
    fill="#DCD8D8"
    fill-rule="evenodd"
  />
  <path
    d="M3.86191 11.1953L1.52858 13.5286C1.26823 13.7889 1.26823 14.2111 1.52858 14.4714C1.78892 14.7318 2.21103 14.7318 2.47138 14.4714L4.80472 12.1381C5.06507 11.8777 5.06507 11.4556 4.80472 11.1953C4.54437 10.9349 4.12226 10.9349 3.86191 11.1953Z"
    fill="#F0F0F0"
  />
  <circle
    cx="9.33335"
    cy="6.66667"
    fill="#F0F0F0"
    r="0.666667"
  />
</svg>
`
